src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -756,8 +756,14 @@ function bp_core_admin_notice_repopulate_blogs_resume() {
  */
 function bp_core_admin_debug_information( $debug_info = array() ) {
 	global $wp_settings_fields;
-	$active_components = array_intersect_key( bp_core_get_components(), buddypress()->active_components );
+
+	if ( ! bp_is_root_blog() ) {
```

Comment:
What's the rationale behind this decision? If I had a network of blogs (BP instances), I'd expect to have data specific to each BP instance in each site, no?

Reply:
Interesting @renatonascalves to me there can only be 1 BP instance no matter the multisite config, as there's only 1 users table and as https://github.com/buddypress/buddypress/blob/master/src/bp-core/bp-core-options.php#L233%7CL234

The multiblog constant is simply displaying pages..There's 1 unique root blog that's why to me it's not needed to display this screen for other blogs that are relying on the unique BP instance.

That being said I may have missed a specific workaround making it possible to have multiple BP instances on a network. Can you light my ignorance about it?

Reply:
So, my recollection of that feature is wrong, then.

https://github.com/buddypress/buddypress/blob/41d7e95513c7c1441fc94e254333be2cb9abb80e/src/class-buddypress.php#L423
https://codex.buddypress.org/getting-started/customizing/bp_enable_multiblog/
https://github.com/buddypress/buddypress/blob/41d7e95513c7c1441fc94e254333be2cb9abb80e/src/bp-core/bp-core-functions.php#L2205

This here makes me think it is possible to have BP in a MU network but being the root of the site in the network.

https://github.com/buddypress/buddypress/blob/41d7e95513c7c1441fc94e254333be2cb9abb80e/src/readme.txt#L87-L90

Could we discuss removing this feature from BP? It seems pretty old, pre-1.7 version.

Reply:
I agree the multiblog mode is confusing. It doesn't mean you can have multiple BP instances. It means you can display the content of the same instance on multiple blogs. The readme text, you referenced is explaining it differently talking about "central data".

> to allow your BuddyPress content to be displayed on any site in your WordPress Multisite network, using the same central data.

I agree we should probably rethink all the possible configs plugin's history made available. Reading the codex page you added a link to seems to explain it can be interesting when you want to use it for translation purposes.

Maybe rethinking the Administration part we've included into our roadmap can help us simplify how BuddyPress can be configured.

Reply:
Agree! Thanks!!